Description: Call gnome_cmd_main_win_update_bookmarks after having bookmarked
 the current directory.
Author: Cristian Buzduga
From: https://bugzilla.gnome.org/attachment.cgi?id=187495&action=diff
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=649375
Bug-Ubuntu: https://bugs.launchpad.net/bugs/743898
Applied-Upstream:
 http://git.gnome.org/browse/gnome-commander/commit/?h=gcmd-1-2-8&id=bf1369d
---
 src/gnome-cmd-dir-indicator.cc |    1 +
 1 file changed, 1 insertion(+)

--- gnome-commander.orig/src/gnome-cmd-dir-indicator.cc
+++ gnome-commander/src/gnome-cmd-dir-indicator.cc
@@ -370,6 +370,7 @@ static void on_bookmarks_add_current (Gt
     bm->group = group;
 
     group->bookmarks = g_list_append (group->bookmarks, bm);
+    gnome_cmd_main_win_update_bookmarks (main_win);
 }
 
 
